Gemfile.lock

```diff
@@ -54,20 +54,20 @@ GEM
       thread_safe (~> 0.3, >= 0.3.4)
```

Comment:
I'm not confident about all this change, may be @arnaudbesnier has an insight about that